app/controllers/concerns/waste_exemptions/after_state_updated.rb

```
@@ -22,5 +22,12 @@ def after_not_known(enrollment, params)
       end
     end
 
+    def after_organisation_address(enrollment, params)
+      enrollment.organisation.contact.addresses.try(:first).try(:update!, address_type: :primary)
```

Do we expect there to be at least 1 address at this point? The reason i ask is if we do, is ```try()``` hiding an exception from us which should actually be raised. Else should we be doing something else if try we're to return ```nil```?